### PR TITLE
Timestamp and format changelog

### PR DESCRIPTION
**Pull request by sandstrom**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
-== 3.5.0
+### 3.5.0 (July 10, 2016)
 * Only add JSON as a dependency if running under Ruby < 2 (#191).
 
-== 3.4.0
+### 3.4.0 (July 8, 2016)
 * Add units support, eg. "12 degrees Celsius", etc.
 
-== 3.3.0
+### 3.3.0 (Mars 29, 2016)
```
Comment (camertron) [inline CHANGELOG.md]: March?

Comment (sandstrom) [inline CHANGELOG.md]: Fixed! I revert to Swedish when I'm tired 😃

**Comment by sandstrom** [comment]
@camertron There aren't git-tags for all versions, e.g. none for `3.1.2`.

I've suggested adding this to make the changelog friendlier to humans. For this it's mostly the top entries that matter (to understand release cadence, etc). I'd suggest just skipping the old entries.

**Comment by camertron** [comment]
Hahaha ok I totally understand! Will merge, thanks :)

**Pull request merged**